### DISCUSSION TITLE
Can a Boolean attribute default to a Indeterminate value/checkbox?

### DISCUSSION THREAD

**Discussion by BusStop101** [atrocore/atrocore#1049] [Q&A] [answered]
We're working on configuring the system and have found that when we configure a product attribute to be of type Boolean it defaults to unchecked/no. As such, there's no way to know if a user has deliberately set it to a no value or they simply haven't reviewed it.
Is it possible to have a boolean value set to indeterminate initially so users are aware it still needs to be set to either a yes or no value? Or is this possible with the Completeness module?

**Comment by AnnZast** [comment, marked as answer]
Hello,
For the attribute of type boolean , there is no way to know whether the value is set to false or not selected. For your case, I would recommend using an attribute of type list with Yes and No options.
